guides/source/configuring.md

```diff
@@ -1153,18 +1153,6 @@ if there were multiple copies of the same record enrolled in the transaction.
 | (original)            | `false`              |
 | 7.1                   | `true`               |
 
-#### `config.active_record.belongs_to_required_by_default`
```

Comment:
I think the argument for deprecating `required:` is sound. But I'm less sure on this config. It wasn't documented as deprecated, and I can see a use for it even in apps today.

Reply:
Hmm I get your point. My thinking was, since it was introduced in https://github.com/rails/rails/pull/18937 specifically to transition to the new default, there's no real use case for it outside of that i.e. if `belongs_to` isn't required, you should probably just set `optional: true` explicitly on each association.